src/metrics/misc.ts

```diff
@@ -194,6 +209,33 @@ export class MetadataMetric implements Metric {
       throw new Error('Invalid result');
     }
 
+    // openings
```

Comment:
better to have comment adding value or none at all

Reply:
good advice ty